console/src/components/Common/FilterQuery/types.ts

```diff
@@ -19,7 +19,11 @@ export type Operator =
   | '$ilike'
   | '$nilike'
   | '$similar'
-  | '$nsimilar';
+  | '$nsimilar'
+  | '$posix_cs'
```

Comment:
Why not call these `$regex`, `$iregex` , `$nregex` and `$inregex` respectively?

Reply:
It makes sense, I was not sure about the name as you can also use SQL regex with SIMILAR.
If you are the one that can take the decision I am perfectly OK to change it now. :)

Reply:
I agree that having "regex" in the name is more descriptive than having "posix" in the name. @christophediprima, please make this change :)

(And thanks for being so responsive!)